apps/docs/content/components/forms/NumberField.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: NumberField
+description: A number field is a specialized number field that allows a user to enter a number.
+category: "forms"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/inputs/src/NumberInputField.tsx
+---
+
+<Example src="inputs/docs/numberField/preview" isOpen />
+
+## Props
+
+<PropTable component="NumberField" />
+
+<FeatureFlag flag="next">
+    ## Guidelines
+
+    TODO: If we have some guidelines about this component&apos;s usage
+
+    ### Accessibility?
+
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
+
+## Anatomy
+
+<FeatureFlag flag="rc">
+### Concepts
+
+TODO: links to related concepts
+</FeatureFlag>
+
+### Composed Components
+
+A `NumberField` uses the following components.
+
+- [Label](./Label)
+- [Helper Message](./HelperMessage)
+- [Error Message](./ErrorMessage)
+
+## Examples
+
+### Disabled
+A number field in a disabled state shows that an input field exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+
+<Example src="inputs/docs/numberField/disabled" />
+
+### ReadOnly
+The `isReadOnly` prop makes the number field's text content immutable. Unlike `isDisabled`, the number field remains focusable and the contents can still be copied. See [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information.
+
+<Example src="inputs/docs/numberField/readOnly" />
+
+### Sizes
+Number fields have multiple sizes to choose from.
+
+<Example src="inputs/docs/numberField/sizes" />
+
+### Labeling
+If a visible label isn't specified, an `aria-label` must be provided to the number field for accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using the ID of the labeling element instead.
+
+<Example src="inputs/docs/numberField/labeling" />
+
+### Description
+A number field with a helper message.
+
+<Example src="inputs/docs/numberField/description" />
+
+### Icon Prefix
+An icon can be displayed at the start of the input.
+
+<Example src="inputs/docs/numberField/iconPrefix" />
+
+### Text Prefix
+A short text can be displayed at the start of the input.
+
+<Example src="inputs/docs/numberField/textPrefix" />
```

Comment:
We seem to be missing the Fluid and Formatting stories.

Reply:
fluid was not in text field, I based myself on the text field documentation page I can add it

Reply:
I'll add error/invalid I can't assume someone will go trough text to figure out its available also in a number input

Reply:
ya but it's not in Text.mdx either. I'm not sure why.